src/main/java/uk/gov/pay/webhooks/app/SqsConfig.java

```diff
@@ -0,0 +1,58 @@
+package uk.gov.pay.webhooks.app;
+
+import io.dropwizard.Configuration;
+
+import javax.validation.constraints.Max;
+import javax.validation.constraints.NotNull;
+
+public class SqsConfig extends Configuration {
+
+    @NotNull
+    private String eventQueueUrl;
+
+    @NotNull
+    private String region;
+    private String accessKey;
+    private String secretKey;
+
+    @Max(20)
+    private int messageMaximumWaitTimeInSeconds;
+    @Max(10)
+    private int messageMaximumBatchSize;
+
+
+    private boolean nonStandardServiceEndpoint;
```

Comment:
I think using language like "AWS environment" or "local environment" might help future us understand this without as much README. No suggestions at this hour, happy for it to be in a follow up after discussion

Reply:
Yeah, let’s change this in all the queue-consuming projects at once.